src/__tests__/Polymesh.ts

```diff
@@ -318,4 +322,133 @@ describe('Polymesh Class', () => {
       expect(result).toMatchObject(new Identity(params, context));
     });
   });
+
+  describe('method: getSecurityTokens', () => {
+    beforeAll(() => {
+      sinon.stub(utilsModule, 'signerToSignatory');
+    });
+
+    afterAll(() => {
+      sinon.restore();
+    });
+
+    test('should return a list of security tokens if did parameter is set', async () => {
+      const fakeTicker = 'TEST';
+
+      polkadotMockUtils.configureMocks({ contextOptions: { withSeed: true } });
+
+      polkadotMockUtils.createQueryStub('identity', 'links', {
+        entries: [
+          [
+            ['someKey'],
+            polkadotMockUtils.createMockLink({
+              // eslint-disable-next-line @typescript-eslint/camelcase
+              link_data: polkadotMockUtils.createMockLinkData({
+                TokenOwned: polkadotMockUtils.createMockTicker(fakeTicker),
+              }),
+              expiry: polkadotMockUtils.createMockOption(),
+              // eslint-disable-next-line @typescript-eslint/camelcase
+              link_id: polkadotMockUtils.createMockU64(),
+            }),
+          ],
+        ],
+      });
+
+      const polymesh = await Polymesh.connect({
+        nodeUrl: 'wss://some.url',
+        accountUri: '//uri',
+      });
+
+      const securityTokens = await polymesh.getSecurityTokens({ did: 'someDid' });
+
+      expect(securityTokens).toHaveLength(1);
+      expect(securityTokens[0].ticker).toBe(fakeTicker);
+    });
+
+    test('should return a list of security tokens owned by the identity', async () => {
```

Comment:
should return a list of security tokens owned by the current identity if no did is supplied